lua/themes/dark-16.lua

```diff
@@ -35,3 +37,123 @@ lexers.STYLE_STATUS_FOCUSED = 'reverse,bold'
 lexers.STYLE_SEPARATOR = lexers.STYLE_DEFAULT
 lexers.STYLE_INFO = 'fore:default,back:default,bold'
 lexers.STYLE_EOF = ''
+
+-- lexer specific styles
+
+-- Diff
+lexers.STYLE_ADDITION = 'fore:green'
+lexers.STYLE_DELETION = 'fore:red'
+lexers.STYLE_CHANGE = 'fore:yellow'
+
+--[[
+-- CSS
+lexers.STYLE_PROPERTY = lexers.STYLE_ATTRIBUTE
+lexers.STYLE_PSEUDOCLASS = ''
+lexers.STYLE_PSEUDOELEMENT = ''
+
+-- HTML
+lexers.STYLE_TAG_UNKNOWN = lexers.STYLE_TAG .. ',italics'
+lexers.STYLE_ATTRIBUTE_UNKNOWN = lexers.STYLE_ATTRIBUTE .. ',italics'
+
+-- Latex, TeX, and Texinfo
+lexers.STYLE_COMMAND = lexers.STYLE_KEYWORD
+lexers.STYLE_COMMAND_SECTION = lexers.STYLE_CLASS
+lexers.STYLE_ENVIRONMENT = lexers.STYLE_TYPE
+lexers.STYLE_ENVIRONMENT_MATH = lexers.STYLE_NUMBER
+
+-- Makefile
+lexers.STYLE_TARGET = ''
+
+-- Markdown
+lexers.STYLE_HR = ''
```

Comment:
The markdown theming example is missing:

* `STYLE_BOLD`
* `STYLE_ITALIC`
* `STYLE_LIST`
* `STYLE_LINK`
* `STYLE_REFERENCE`
* `STYLE_HEADING_H1` - `STYLE_HEADING_H6`

Here is what I use in my theme:


```lua
lexers.STYLE_BOLD = 'fore:default,back:default,bold'
lexers.STYLE_ITALIC = 'fore:default,back:default,italics'
lexers.STYLE_LIST = lexers.STYLE_KEYWORD
lexers.STYLE_LINK = lexers.STYLE_KEYWORD
lexers.STYLE_REFERENCE = lexers.STYLE_KEYWORD
lexers.STYLE_HR = 'fore:red'
for i=1, 6 do lexers['STYLE_HEADING_H'..i] = 'fore:red' end
```

Reply:
Thanks! The `STYLE_HEADING_H?` are already in the `txt2tags` example but I can add your line to both. As for the others I missed them since they were defined elsewhere in the scintillua themes. They are also used in a couple other lexers so it might be useful to just define them by default.

By the way stuff like:
```lua
lexers.STYLE_BOLD = 'fore:default,back:default,bold'
```
is equivalent to:
```lua
lexers.STYLE_BOLD = 'bold'
```

Reply:
Oops the `txt2tags` is `STYLE_H?` (which from what I see in the lexer is never actually used). Either way both can be replaced by the `for` loop.

Reply:
> Oops the `txt2tags` is `STYLE_H?` (which from what I see in the lexer is never actually used). Either way both can be replaced by the `for` loop.

It is actually used at: https://github.com/martanne/vis/blob/b753520d1f4adcf752e2b3f882d64313f0b2120e/lua/lexers/txt2tags.lua#L24

I have not confirmed it but looking at the code the tokens `h1` up to `h6` are defined.